Cargo.toml

```diff
@@ -758,7 +756,6 @@ debug-assertions = true
 debug = true
 
 [patch.crates-io]
-serde-reflection = { git = "https://github.com/aptos-labs/serde-reflection", rev = "839aed62a20ddccf043c08961cfe74875741ccba" }
```

Comment:
I think we did this to ensure that the Move repo used this as well?

Reply:
I see. It wasn't overriding actually, so I thought it wasn't needed. Fixed it now by pointing dependency in `move-bytecode-utils` crate to workspace.

Thanks for the stamp. I need to land this first: https://github.com/aptos-labs/aptos-indexer-processors/pull/218.